src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class StorageDaoImpl implements StorageDao {
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd_HH-mm-ss");

```

Comment:
make data time formatter the constant field.
```suggestion
    private static final DateTimeFormatter FORMATTED = DateTimeFormatter.ofPattern("yyyy-MM-dd_HH-mm-ss");

```